org.dharkael.Flameshot.desktop

```diff
@@ -0,0 +1,48 @@
+[Desktop Entry]
```

Comment:
You are not using the file in the manifest, so I assume upstream's one is used. Please contribute your changes there, if any.